base.Dockerfile

```diff
@@ -118,7 +118,8 @@ ENV PATH=${TMPDIR_FOR_RUBY_WRAPPERS_DIR}:${PATH}
 # Install node.js, yarn and other runtime dependencies.
 COPY --from=builder /usr/share/keyrings/nodesource.gpg /usr/share/keyrings/
 
-RUN install_packages ca-certificates curl libjemalloc-dev libgdbm6 libyaml-0-2 \
+RUN install_packages ca-certificates curl libjemalloc-dev libgdbm6 \
+      libjsonnet0 libyaml-0-2 \
```

Comment:
Optional: is it worth sticking in comment as reminder to check if we can remove

Reply:
Hm good question. Since the gem tries to define [its own build process for libjsonnet++](https://github.com/yugui/ruby-jsonnet/blob/e8c4469/ext/jsonnet/extconf.rb#L18) instead of building it properly, it's probably the lesser of evils to stick with the Ubuntu package even if the ruby-jsonnet maintainers get back on it. Worst case is the API changes and we end up having to pin the Apt package version, which is still better than being stuck waiting on absent third-party gem maintainers.

What you reckon?

Reply:
Yup make sense!